docs/conf.py

```diff
@@ -12,8 +12,8 @@
 # -- Project information -----------------------------------------------------
 
 project = "hermes_eea"
-copyright = "2021, Jane Doe"
-author = "Jane Doe"
+copyright = "No rights reservers"
```

Comment:
Is reservers a typo

Reply:
It is a typo. Thanks for catching that. I actually just read through an NPR that suggests that we should add a copyright notice even though as a work of the US government we can't claim copyright protection.